.github/workflows/nano_unit_tests_pytorch.yml

```diff
@@ -24,10 +24,10 @@ jobs:
         os: ["ubuntu-20.04"]
         python-version: ["3.7"]
         pytorch-version: [
-          "torch==1.9.0+cpu torchvision==0.10.0+cpu",
-          "torch==1.10.1+cpu torchvision==0.11.2+cpu",
-          "torch==1.12.0+cpu torchvision==0.13.0+cpu",
-          ""
+          "torch==1.9.0+cpu torchvision==0.10.0+cpu intel_extension_for_pytorch==1.11.0",
```

Comment:
Does ipex 1.11.0 works with all previous version?

Reply:
Yes, and I think this workflow has used ipex1.11.0 before to adapt to torch 1.9-1.11.

https://github.com/intel-analytics/BigDL/blob/685ec6c494df389564cf4fa6bc514fde35169574/python/nano/setup.py#L80-L88

https://github.com/intel-analytics/BigDL/blob/685ec6c494df389564cf4fa6bc514fde35169574/.github/workflows/nano_unit_tests_pytorch.yml#L27-L30

Reply:
I see. 

I think this is because this workflow only test pytorch-only test cases and no ipex cases. 
It may not work if if you call ipex 1.11.0 using pytorch 1.10 and before.

Reply:
Thanks, I know what you mean. I haven't specially tested this. (Due to network reasons) I will merge this PR first, and then open a separate PR to test the support of ipex version.